docs/auth.md

```diff
@@ -0,0 +1,259 @@
+--wrap=0 <file>`.  The resulting output may be passed to the Helm charts
+for the `apiserver.tls.*` series of options.
+
+### Serving Certificates
+
+In order to securely serve the service catalog API server over HTTPS,
+you'll need serving certificates.  By default, a set of self-signed
+certificates are generated by the service catalog API server. However,
+client have no way to trust these (since they are self-signed, there is no
+separate CA), so production deployments, or deployments running behind an
+API server aggregator, should use manually generated CA certificates.
+
+By default, the service catalog API server looks for these certificates in
+the `/var/run/kubernetes-service-catalog` directory, although this may be
+overridden using the `--cert-dir` option on the API server.  The files
+must be named `apiserver.crt` and `apiserver.key`.
+
+When deploying using the Helm charts, you should pass the
+`apiserver.tls.cert` and `apiserver.tls.key` options, populated with the
+contents of the aforementioned files, base64 encoded.
+
+## Authentication
+
+The service catalog API server makes use of the standard Kubernetes add-on
+API server delegated authentication setup.  There are several components
+of the delegated authentication setup, detailed below.
+
+### Client Certificate Authentication
+
+Client certificate authentication authenticates clients who connect using
+certificates signed by a given CA (as specified by the *client CA
+certificate*).  This same mechanism is also generally used by the main
+Kubernetes API server.
+
+Generally, the default admin user in a cluster connects with client
+certificate authentication.  Additionally, off-cluster non-human clients
+often use client certificate authentication.
+
+By default, a main Kubernetes API server configured with the
+`--client-ca-file` option automatically creates a ConfigMap called
+`extension-apiserver-authentication` in the `kube-system` namespace,
+populated with the the client CA file.  The service catalog API server use
+this CA certificate as the CA used to verify client authentication. This
+way, client certificate users who can authenticate with the main
+Kubernetes system can also authenticate with the service catalog API
+server.
+
+See the [delegated token authentication](#delegated-token-authentication)
+section for more information about how the service catalog API server
+contacts the main Kubernetes API server to access this ConfigMap.
+
+If you wish to use a different client CA certificate to verify client
+certificate authentication, you can manually pass the `--client-ca-file`
+option to the service catalog API server.
+
+See the [x509 client
+certificates](https://kubernetes.io/docs/admin/authentication/#x509-client-certs)
+section of the Kubernetes documentation for more information.
+
+### Delegated Token Authentication
+
+Delegated token authentication authenticates clients who pass in a token
```

Comment:
a couple things describe similar things, but, IIRC none of the Kubernetes docs actually describe *delegating* token auth.  I'll see what I can dig up.